src/System.IO.Compression/src/Interop/Interop.zlib.Windows.cs

```diff
@@ -52,6 +52,23 @@ internal static partial class zlib
             }
         }
 
+        internal static unsafe bool IsCrc32Available()
+        {
+            try
+            {
+                // Make a P/Invoke into zlib crc32 to ensure we're able to find and use it.
+                // If we are, then use zlib.
+                crc32(0, null, 0);
+                return true;
+            }
```

Comment:
I did a check in TFS, and the desktop .csproj file only includes Crc32Helper.cs (where it calls into this method), and not this Interop file.
I believe @ellismg was the one who included the #if to fix the build break in the desktop build. He can verify that we are safe here.